### PR TITLE
Fix 404s

### PR DESCRIPTION
**Pull request by dmundra**
Fixed some 404s after running https://www.brokenlinkcheck.com/

Noticed that https://alphagov.github.io/accessibility-guidance/ was taken down (repo: https://github.com/alphagov/accessibility-guidance)

### CODE REVIEW

File: news/index.html [hunk 1]
```diff
@@ -2,9 +2,11 @@
 title: News
 layout: page
 description: Latest CivicActions Accessibility news.
+redirect_from:
+- /posts/
```
Comment (dmundra) [inline news/index.html:6]: This fixes the breadcrumb link for posts. Also the remark-link in this file can be ignored.

File: _playbook/training.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ Some of our favorite, free accessibility courses include:
...
 
-There are also several high-quality accessibility guides available for free. The GDS has produced [accessibility guides](https://alphagov.github.io/accessibility-guidance/), as has the US government's [18F](https://accessibility.18f.gov/). Rather than creating a new guide from scratch, these resources are available to build upon and improve. CivicActions is looking to extend these to provide an [accessibility playbook](https://civicactions.github.io/accessibility/) for people interested in accessibility best practices.
```
Comment (mikang) [inline _playbook/training.md:23]: Maybe it should be "the US government has produced a guide with 18F"? Because 18F is an organization, not resources.

**Review by mikang — CHANGES_REQUESTED** [review]
Looks great! I added a comment to one of the changes. Let me know if you can't see it.

File: _playbook/training.md (hunk 2 above)
Comment (dmundra) [reply _playbook/training.md:23]: @mikang good call. That makes more sense. Pushed a change

**Review by mikang — APPROVED** [review]
Good to go!

**Pull request merged**